foxsec_sqs_worker.py

```diff
@@ -0,0 +1,231 @@
+"""FoxSec SQS worker"""
+import os
+import datetime
+import functools
+import time
+import json
+import ipaddress
+import boto3
+from botocore.exceptions import ClientError
+from pprint import pprint
+
+# Config
+# region = os.environ.get('AWS_DEFAULT_REGION', 'us-east-1')
+WAF_IPSET_ID = os.environ.get('IPSET_ID')
+WAF_IPSET_EXPIRATION_HOURS = os.environ.get('IPSET_EXPIRATION_HOURS', 24)
+
+# Constants
+DYNAMODB = boto3.resource('dynamodb').Table('foxsec-waf')
+SQS = boto3.client('sqs')
+WAFREGIONAL = boto3.client('waf-regional')
+
+
+def parse_arn(arn):
```

Comment:
Can this ever be sent an invalid ARN? There doesn't seem to be exception handling for that.